Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /app
 COPY . /app
 
 RUN python -m venv /app
-RUN ./bin/pip install -r requirements-dev.txt
+RUN ./bin/pip install -r requirements/base.txt
```

Comment:
++ for this naming. Makes more sense to have `base` + `test` rather than existing naming.

Reply:
\o/